Maquetado/index.html

```diff
@@ -11,14 +11,79 @@
 </head>
 
 <body>
+  
   <h2 class="title">Front-End Tools</h2>
-
-  <div class="tech">
-    <img class="img-tech" src="assets/react_icon.svg" alt="Logo React">
-    <h3 class="title-tech">ReactJS</h3>
-    <span class="release-tech">Release: 2013</span>
-    <p class="txt-tech">React is a declarative, component-based Javascript library for building user interfaces.</p>
+  
+  <div class="row">
```

Comment:
use just a single div to wrap all the techs. use the flex-wrap propery to make it wrap instead of shrink.